canonical-websites/www.ubuntu.com
# Move UASD to markdown

## Done

* converted the /legal/ubuntu-advantage-service-description page to use markdown

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the existing live page, see that only the internal links are highlighted and that all copy is the same.
